src/quo2/components/numbered_keyboard/numbered_keyboard/style.cljs

```diff
@@ -0,0 +1,18 @@
+(ns quo2.components.numbered-keyboard.numbered-keyboard.style)
+
+(def container
+  {:flex-direction     :row
+   :flex-wrap          :wrap
+   :flex               1
+   :padding-horizontal 34
+   :padding-top        8
+   :padding-bottom     0})
+
+(defn keyboard-item
+  [position]
```

Comment:
nit: an alternative would be `column`

Reply:
It's related to column. I will rename it to column-position

Reply:
Removed